README.md

```diff
@@ -2,7 +2,6 @@
 
 ![Download count all time](https://img.shields.io/npm/dt/ember-cli-flash.svg) [![npm version](https://badge.fury.io/js/ember-cli-flash.svg)](http://badge.fury.io/js/ember-cli-flash)
 [![Ember Observer Score](http://emberobserver.com/badges/ember-cli-flash.svg)](http://emberobserver.com/addons/ember-cli-flash)
-[![Code Climate](https://codeclimate.com/github/adopted-ember-addons/ember-cli-flash/badges/gpa.svg)](https://codeclimate.com/github/adopted-ember-addons/ember-cli-flash)
```

Comment:
Is 404 
<img width="190" alt="image" src="https://github.com/adopted-ember-addons/ember-cli-flash/assets/36919/019125ae-49df-4ee3-8026-ba5df364bcf9">